packages/financial-templates-lib/src/price-feed/CryptoWatchPriceFeed.js

```diff
@@ -66,7 +66,11 @@ class CryptoWatchPriceFeed extends PriceFeedInterface {
   }
 
   getCurrentPrice() {
-    return this.invertPrice ? this._invertPriceSafely(this.currentPrice) : this.currentPrice;
+    return this.twapLength
+      ? this.currentPrice
+      : this.invertPrice
+      ? this._invertPriceSafely(this.currentPrice)
+      : this.currentPrice;
```

Comment:
I think nested ternaries are a little difficult to read. Do you think this would be a little easier to understand?
```js
if (!this.twapLength && this.invertPrice) {
  // Maybe add a comment here as to why we need to do this for this case?
  return this._invertPriceSafely(this.currentPrice);
} else {
  return this.currentPrice;
}
```
  
Thoughts? Do you agree with this logic?

Reply:
I do agree with the logic and that this is more readable. Updated to use this instead!